C++/syntax_test_cpp.cpp

```diff
@@ -2886,6 +2886,39 @@ export import ABC:d
 /*               ^  punctuation.accessor.c++ */
 /*                ^ entity.name.partition.other */
 
+export module .test.module;
+/* <- meta.module.c++ keyword.control.module.c++ */
+/*     ^^^^^^ keyword.control.module.c++ */
+/*            ^ invalid.illegal.unexpected-character.c++ */
+/*             ^^^^^^^^^^^ source.c++ */
```

Comment:
You might want to use negative tests to check that it is not matched as module name.

```suggestion
/*             ^^^^^^^^^^^ - keyword - entity */
```

Reply:
Oh wow, the ST documentation doesn't even mention negative tests.  That's pretty useful.